testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDeleteCompatibility.java

```diff
@@ -427,8 +447,7 @@ public void testDeletionVectorsAcrossAddFile(boolean partitioned)
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
-    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
```

Comment:
Only running with DELTA_LAKE_OSS doesn't give helpful test coverage:
```java
    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
    public void testDeletionVectorsTruncateTable()
    {
        testDeletionVectorsDeleteAll(tableName -> {
            if (databricksRuntimeVersion.isPresent()) {
                onDelta().executeQuery("TRUNCATE TABLE default." + tableName);
            }
            else {
                assertThatThrownBy(() -> onDelta().executeQuery("TRUNCATE TABLE default." + tableName))
                        .hasMessageContaining("Table does not support truncates");
                throw new SkipException("OSS Delta Lake doesn't support truncating tables with deletion vector");
            }
        });
    }
```

Reply:
I noticed it the first time i was doing changes, but forgot yesterday about it.